exe/wallet/Main.hs

```diff
@@ -245,24 +246,28 @@ exec execServer manager args
     | args `isPresent` command "wallet" &&
       args `isPresent` command "delete" = do
         wId <- args `parseArg` argument "wallet-id"
-        runClient (const "") $ deleteWallet (ApiT wId)
+        runClient (const "") (deleteWallet (ApiT wId))
 
     | args `isPresent` command "transaction" &&
       args `isPresent` command "create" = do
         wId <- args `parseArg` argument "wallet-id"
         ts <- args `parseAllArgs` longOption "payment"
-        wPwd <- getPassphrase
-        runClient Aeson.encodePretty $ createTransaction (ApiT wId) $
-            PostTransactionData
-                ts
-                (ApiT wPwd)
+        res <- sendRequest $ getWallet $ ApiT wId
+        if (isRight res) then do
```

Comment:
Just a minor thing but pattern-matching `case res of Right _; Left _;` is more idiomatic in Haskell than `if isXXX then else`